gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -123,7 +123,7 @@ ext.externalDependency = [
     "influxdbJava": "org.influxdb:influxdb-java:2.1",
     "kryo": "com.esotericsoftware.kryo:kryo:2.22",
     "libthrift":"org.apache.thrift:libthrift:0.9.3",
-    "lombok":"org.projectlombok:lombok:1.16.8",
+    "lombok":"org.projectlombok:lombok:1.18.10",
```

Comment:
Do you think we can bump to 1.18.16? This version can resolve an IntelliJ incompatibility issue.

FYI: https://stackoverflow.com/questions/65128763/java-you-arent-using-a-compiler-supported-by-lombok-so-lombok-will-not-work-a

Reply:
Updated